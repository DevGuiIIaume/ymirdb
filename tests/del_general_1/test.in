set a 1
set b a
set c b
set d a b c
list entries
forward a
forward b
forward c
forward d
backward a
backward b
backward c
backward d
del a
del b
del c
del d
list entries
forward a
forward b
forward c
forward d
backward a
backward b
backward c
backward d
del a
del b
del c
list entries
get c
forward a
forward b
backward a
backward b
del a
del b
forward a
forward b
backward a
backward b
list entries
del a
forward a
backward a
bye
