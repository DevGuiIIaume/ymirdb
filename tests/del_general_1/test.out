> ok

> ok

> ok

> ok

> d [a b c]
c [b]
b [a]
a [1]

> nil

> a

> a, b

> a, b, c

> b, c, d

> c, d

> d

> nil

> not permitted

> not permitted

> not permitted

> ok

> c [b]
b [a]
a [1]

> nil

> a

> a, b

> no such key

> b, c

> c

> nil

> no such key

> not permitted

> not permitted

> ok

> b [a]
a [1]

> no such key

> nil

> a

> b

> nil

> not permitted

> ok

> nil

> no such key

> nil

> no such key

> a [1]

> ok

> no such key

> no such key

> bye
